src/realm/object-store/set.cpp

```diff
@@ -284,4 +294,68 @@ std::pair<size_t, bool> Set::insert<Obj>(Obj obj)
     return as<ObjKey>().insert(obj.get_key());
 }
 
+bool Set::is_subset_of(const Set& rhs) const
```

Comment:
Should these be `Collection&` instead? We should be able to call `Set::is_subset_of(some_list)`.

Reply:
I see those are not yet exposed by Core - I filed https://github.com/realm/realm-core/issues/4226 to track that.